clients/unshaded/src/main/java/tachyon/client/file/policy/MostAvailableFirstPolicy.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.file.policy;
+
+import java.util.List;
+
+import com.google.common.collect.Lists;
+
+import tachyon.client.block.BlockWorkerInfo;
+
+/**
+ * A policy that returns the worker with the most available bytes. The policy returns null if no
```

Comment:
ah, good to know it. From the user perspective, this policy makes sense. However, I understand currently there's no way to get up-to-date *real* used bytes on workers. I think an improvement on master to reflect that state is needed.